### ISSUE TITLE
MTL platform with ARC 770 cannot allocate memory block with size lager than 4GB when running vLLM Qwen2-VL-2B

### ISSUE THREAD

**Issue by weijiejx** [intel-analytics/ipex-llm#12136]
when I run vLLM model like Qwen2-VL-2B with ARC770 on MTL platform, will report error message as below:
RuntimeError: Current platform can NOT allocate memory block with size larger than 4GB! Tried to allocate 6.10 GiB (GPU  0; 15.11 GiB total capacity; 4.84 GiB already allocated; 5.41 GiB reserved in total by PyTorch)

Here is example code for running Qwen2-VL-2B:

from transformers import Qwen2VLForConditionalGeneration, AutoTokenizer, AutoProcessor
from qwen_vl_utils import process_vision_info
# from modelscope import snapshot_download
![Screenshot from 2024-09-27 15-45-34](https://github.com/user-attachments/assets/c7304567-3b81-4874-8edf-b81cc8fdc72d)
![Screenshot from 2024-09-27 15-45-34](https://github.com/user-attachments/assets/835941b7-5943-411c-8f64-a873ea2cdc8a)

# model_dir = snapshot_download("qwen/Qwen2-VL-2B-Instruct")
model_dir = "/home/ugreen/Downloads/Qwen2-VL-2B-Instruct"

# default: Load the model on the available device(s)
model = Qwen2VLForConditionalGeneration.from_pretrained(
    model_dir, torch_dtype="auto", device_map="**xpu:0**"
)

# We recommend enabling flash_attention_2 for better acceleration and memory saving, especially in multi-image and video scenarios.
# model = Qwen2VLForConditionalGeneration.from_pretrained(
#     model_dir,
#     torch_dtype=torch.bfloat16,
#     attn_implementation="flash_attention_2",
#     device_map="auto",
# )

# default processer
processor = AutoProcessor.from_pretrained(model_dir)

# The default range for the number of visual tokens per image in the model is 4-16384. You can set min_pixels and max_pixels according to your needs, such as a token count range of 256-1280, to balance speed and memory usage.
# min_pixels = 256*28*28
# max_pixels = 1280*28*28
# processor = AutoProcessor.from_pretrained("Qwen/Qwen2-VL-2B-Instruct", min_pixels=min_pixels, max_pixels=max_pixels)

messages = [
    {
        "role": "user",
        "content": [
            {
                "type": "image",
                "image": "https://qianwen-res.oss-cn-beijing.aliyuncs.com/Qwen-VL/assets/demo.jpeg",
            },
            {"type": "text", "text": "Describe this image."},
        ],
    }
]

# Preparation for inference
text = processor.apply_chat_template(
    messages, tokenize=False, add_generation_prompt=True
)
image_inputs, video_inputs = process_vision_info(messages)
inputs = processor(
    text=[text],
    images=image_inputs,
    videos=video_inputs,
    padding=True,
    return_tensors="pt",
)
inputs = inputs.to("xpu:0")

# Inference: Generation of the output
generated_ids = model.generate(**inputs, max_new_tokens=128)
generated_ids_trimmed = [
    out_ids[len(in_ids) :] for in_ids, out_ids in zip(inputs.input_ids, generated_ids)
]
output_text = processor.batch_decode(
    generated_ids_trimmed, skip_special_tokens=True, clean_up_tokenization_spaces=False
)
print(output_text)

**Comment by hzjane** [comment]
vllm 0.5.4 does not support qwen2-vl model yet. We will support it in the future 0.6.1 version.

**Comment by weijiejx** [comment]
Thank you! But I need double confirm, I use ipex to run Qwen2-VL-2B, not OpenVINO, vLLM 0.5.4 not support, right?

**Comment by hzjane** [comment]
Yes, even the official version of vllm 0.5.4 does not support it until 0.6.1.

**Comment by weijiejx** [comment]
Thanks again.
One more question, is any vLLM model available that I can use with vllm 0.5.4? Can you advise me one or two that I can try it.
Thanks.

**Comment by hzjane** [comment]
It is recommended to run Llama Qwen and chatglm models. 
for example: `Llama-2-7b-chat-hf Qwen1.5-7B-Chat chatglm3-6b`.

**Comment by weijiejx** [comment]
Hi，
Could you kindly point out what schedule of 0.6.1 for release if you know it, thanks.

**Comment by glorysdj** [comment]
We are validating 0.6.2 version, and Qwen2-VL model, will notify you once it's ready. Thanks.
